src/main/java/codeu/controller/SearchServlet.java

```diff
@@ -65,18 +66,24 @@ void setMessageStore(MessageStore messageStore) {
   public void doGet(HttpServletRequest request, HttpServletResponse response) 
       throws IOException, ServletException {
     String searchRequest = (String)request.getParameter("searchRequest");
-    request.setAttribute("searchRequest", searchRequest);
 
+    // TODO 
+    if (searchRequest == null) {
+
+    }
+    
     List<User> users = userStore.getUsers(searchRequest);
-    request.setAttribute("users", users);
+    
+    String json = new Gson().toJson(users);
+    response.setContentType("application/json");
+    response.setCharacterEncoding("UTF-8");
+    response.getWriter().write(json);
 
     // List<Conversation> conversations = conversationStore.getAllConversations();
```

Comment:
Can we clean up these comments? If there's anything you want to remember to do, add a `// TODO` line instead.